requirements.txt

```
@@ -0,0 +1,2 @@
+ConfigArgParse
```

```suggestion
ConfigArgParse~=0.11.0
```

To clarify that it works with the version [from CentOS 7](https://centos.pkgs.org/7/epel-x86_64/python2-configargparse-0.11.0-1.el7.noarch.rpm.html). Preferably we have versions in all of these to document what was supported at the time of writing.